test/TEST-01-BASIC/test.sh

```diff
@@ -5,7 +5,9 @@ IMAGE_NAME="basic"
 RUN_IN_UNPRIVILEGED_CONTAINER=${RUN_IN_UNPRIVILEGED_CONTAINER:-yes}
 TEST_REQUIRE_INSTALL_TESTS=0
 
-. $TEST_BASE_DIR/test-functions
+. $(dirname ${BASH_SOURCE[0]})/../test-functions
```

Comment:
is there any way we can avoid this change?

Reply:
it adds quite a bit of churn, and shouldn't be necessary, right?

Reply:
is your objection to removing $TEST_BASE_DIR or using ${BASH_SOURCE[0]}?

Reply:
> it adds quite a bit of churn, and shouldn't be necessary, right?

I don't get what you mean, it should evaluate to exactly the same path? Or do you just mean churn as far as code change?

Reply:
yes, sorry, I meant that the diff is rather large and touches all test scripts just for that, and it's a bit less readable - but if it can't be avoided it's fine

Reply:
having said that, I really don't want to block progress in improving the autopkgtest suite, so if this is the better way, let's go for it

Reply:
TBH it's been a month or so since I coded this up so I don't *think* the TEST_BASE_DIR change is needed but I don't remember completely; I don't have any strong feelings either way (it just seemed cleaner to me) but I'm happy to pull that commit out.

Reply:
ok the commit to remove TEST_BASE_DIR from the test.sh scripts is removed in the latest push